<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Hatem Hunish</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>920</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Untitled-12-512.png</normaloff>Untitled-12-512.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#222;
  border-radius:7px;
  box-shadow: 0px 3px 10px 1px rgba(0, 0, 0, 0.9);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1051</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>87</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">


  background:#161616;
  border:none;
  box-shadow: 0px 1px 0px 0px #111;
  border-radius:3px;
  outline:none;
  color:#FFBF00;
  font-weight:700;
  letter-spacing:0.125em;
  text-align:center;
  text-transform:uppercase;</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_9">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>87</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">


  background:#161616;
  border:none;
  box-shadow: 0px 1px 0px 0px #111;
  border-radius:3px;
  outline:none;
  color:#FFBF00;
  font-weight:700;
  letter-spacing:0.125em;
  text-align:center;
  text-transform:uppercase;</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>87</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">


  background:#161616;
  border:none;
  box-shadow: 0px 1px 0px 0px #111;
  border-radius:3px;
  outline:none;
  color:#FFBF00;
  font-weight:700;
  letter-spacing:0.125em;
  text-align:center;
  text-transform:uppercase;</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_10">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>87</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">


  background:#161616;
  border:none;
  box-shadow: 0px 1px 0px 0px #111;
  border-radius:3px;
  outline:none;
  color:#FFBF00;
  font-weight:700;
  letter-spacing:0.125em;
  text-align:center;
  text-transform:uppercase;</string>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>87</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">


  background:#161616;
  border:none;
  box-shadow: 0px 1px 0px 0px #111;
  border-radius:3px;
  outline:none;
  color:#FFBF00;
  font-weight:700;
  letter-spacing:0.125em;
  text-align:center;
  text-transform:uppercase;</string>
       </property>
       <property name="text">
        <string>Show All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_11">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>87</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">


  background:#161616;
  border:none;
  box-shadow: 0px 1px 0px 0px #111;
  border-radius:3px;
  outline:none;
  color:#FFBF00;
  font-weight:700;
  letter-spacing:0.125em;
  text-align:center;
  text-transform:uppercase;</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_12">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>87</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">


  background:#161616;
  border:none;
  box-shadow: 0px 1px 0px 0px #111;
  border-radius:3px;
  outline:none;
  color:#FFBF00;
  font-weight:700;
  letter-spacing:0.125em;
  text-align:center;
  text-transform:uppercase;</string>
       </property>
       <property name="text">
        <string>Print</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>1051</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="Name_I">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#2c2c2c;
  border:none;
  box-shadow: 0px 1px 0px 0px #111;
  border-radius:3px;
  outline:none;
  color:#FFBF00;
  font-weight:700;
  letter-spacing:0.125em;
  text-align:center;
  text-transform:uppercase;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Meal_I">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#2c2c2c;
  border:none;
  box-shadow: 0px 1px 0px 0px #111;
  border-radius:3px;
  outline:none;
  color:#FFBF00;
  font-weight:700;
  letter-spacing:0.125em;
  text-align:center;
  text-transform:uppercase;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Meal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Price_I">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#2c2c2c;
  border:none;
  box-shadow: 0px 1px 0px 0px #111;
  border-radius:3px;
  outline:none;
  color:#FFBF00;
  font-weight:700;
  letter-spacing:0.125em;
  text-align:center;
  text-transform:uppercase;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Price</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Pay_I">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#2c2c2c;
  border:none;
  box-shadow: 0px 1px 0px 0px #111;
  border-radius:3px;
  outline:none;
  color:#FFBF00;
  font-weight:700;
  letter-spacing:0.125em;
  text-align:center;
  text-transform:uppercase;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Payment</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>1051</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="styleSheet">
        <string notr="true">background:#2c2c2c;
  border:none;
  box-shadow: 0px 1px 0px 0px #111;
  border-radius:3px;
  outline:none;
  color:#FFBF00;
  font-weight:700;
  letter-spacing:0.125em;
  text-align:center;
  text-transform:uppercase;</string>
       </property>
       <property name="plainText">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Note</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>1051</width>
      <height>571</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date/Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Meal</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Payment</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1051</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ae_AlManzomah</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>مطعم يثرب</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="minbtn">
       <property name="styleSheet">
        <string notr="true">box-sizing: border-box;
  appearance: none;
  background-color: transparent;
  border: 2px solid #FFCB3D;
  border-radius: 0.6em;
  color: #FFCB3D;
  cursor: pointer;
  display: flex;
  align-self: center;
  font-size: 1rem;
  font-weight: 400;
  line-height: 1;

  text-decoration: none;
  text-align: center;
  text-transform: uppercase;
  font-family: 'Montserrat', sans-serif;
  font-weight: 700;</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitbtn">
       <property name="styleSheet">
        <string notr="true">box-sizing: border-box;
  appearance: none;
  background-color: transparent;
  border: 2px solid red;
  border-radius: 0.6em;
  color: red;
  cursor: pointer;
  display: flex;
  align-self: center;
  font-size: 1rem;
  font-weight: 400;
  line-height: 1;

  text-decoration: none;
  text-align: center;
  text-transform: uppercase;
  font-family: 'Montserrat', sans-serif;
  font-weight: 700;</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1072</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
